PTN/patterns.py

```diff
@@ -350,6 +351,20 @@
     + patterns["subtitles"][-2]
     + ")",
 ]
+
+patterns["edition"] = [
+    (r"\b\d{{2,3}}(th)?{d}Anniversary{d}(Edition|Ed)?\b".format(d=delimiters), "Anniversary Edition"),
+    (r"\bUltimate{d}Edition\b".format(d=delimiters), "Ultimate Edition"),
+    (r"\bExtended{d}Director\"?s\b".format(d=delimiters), "Directors Cut"), # Needs to be before "Extended"
+    (r"\bExtended\b", "Extended Edition"),
+    (r"\bDirector\"?s{d}Cut\b".format(d=delimiters), "Directors Cut"),
```

Comment:
you may combine the "Directors Cut"

Reply:
I can fix this tonight, thanks! Did you just want it on one line separated with a pipe? Or just cover `director` instead of both edge cases?

Reply:
one line separated with a pipe